### PR TITLE
Spreadsheet: Prevent infinite loop in Range ctor

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

File: Base/res/js/Spreadsheet/runtime.js [hunk 1]
```diff
@@ -162,6 +162,9 @@ class Ranges {
 
 class Range {
     constructor(startingColumnName, endingColumnName, startingRow, endingRow, columnStep, rowStep) {
+        // have to account for '0' since js will parse `+'0'` to 0
+        if (columnStep === 0 || columnStep === '0' || rowStep === 0 || rowStep === '0')
```
Comment (Smrtnyk) [inline Base/res/js/Spreadsheet/runtime.js:166]: I think you can also do
`if (columnStep == '0' || rowStep == '0')` if you want to have it shorter, but I guess strict checks are nicer

Comment (alimpfard) [inline Base/res/js/Spreadsheet/runtime.js:166]: How about just `columnStep == 0`?

**Comment by alimpfard** [comment]
(also, pls run prettier)

File: Base/res/js/Spreadsheet/runtime.js (hunk 1 above)
Comment (u9g) [reply Base/res/js/Spreadsheet/runtime.js]: done

Comment (u9g) [reply Base/res/js/Spreadsheet/runtime.js]: yep, done

**Comment by u9g** [comment]
> (also, pls run prettier)

done :)

**Pull request merged**